### PR TITLE
Implemented the Embedding of Intentions in Actor

### PR DESCRIPTION
**Pull request by Domily11**
Took out embedBasicActior and determined that intentionID is not needed as a list of embedded cells are automatically generated

### CODE REVIEW

File: leaf-ui/js/onFunctions.js [hunk 1]
```diff
@@ -891,22 +891,32 @@ paper.on({
...
                             var userIntention = model.getIntentionByID(cell.attributes.nodeID);
                             if (userIntention.nodeActorID !== '-') {
                                 var actor = model.getActorByID(userIntention.nodeActorID);
                                 actor.removeIntentionID(userIntention.nodeID);
                             }
+                            */
                         }
                         // embed element in new actor
-                        embedBasicActor(cell);
+                        var overlapCells = paper.findViewsFromPoint(cell.getBBox().center());
```
Comment (leshghi) [inline leaf-ui/js/onFunctions.js:879]: Can you reset the IntentionBBM actorID variable to '-' if overlapCells.length = 0

**Review by leshghi — CHANGES_REQUESTED** [review]
Looks good to me - respond to comment and pull from backbone-refactor again and you should be set!

File: leaf-ui/js/onFunctions.js (hunk 1 above)
Comment (Domily11) [reply leaf-ui/js/onFunctions.js:879]: You don't need it tho. cell.get('parent') automatically gets the node the cell is embedded to

Comment (Domily11) [reply leaf-ui/js/onFunctions.js:879]: If it is unembedded the function doesn't return the parent

Comment (leshghi) [reply leaf-ui/js/onFunctions.js:879]: Oh awesome - should we remove the ActorId param from intentions then?

Comment (Domily11) [reply leaf-ui/js/onFunctions.js:879]: I think so

**Review by leshghi — APPROVED** [review]
Looks good for just merging into another one of your branches. Will do a more comprehensive review for mack-actorinspector PR

**Pull request merged**